sqllineage/config.py

```diff
@@ -1,28 +1,77 @@
 import os
+from dataclasses import dataclass
+from typing import Any, Union
+
+
+@dataclass
+class SQLLineageConfigValue:
+    class_type: Any
+    value: Union[str, bool, None]
+
+
+@dataclass
+class SQLLineageConfigDef:
+    DIRECTORY: SQLLineageConfigValue
+    DEFAULT_SCHEMA: SQLLineageConfigValue
+    TSQL_NO_SEMICOLON: SQLLineageConfigValue
```

Comment:
Make these two dataclass private as well. Only expose instance SQLLineageConfig from this module.